src/mixins/filter-mixin.js

```diff
@@ -15,17 +15,56 @@ export function addFilterHandler(filters, filter) {
   return filters
 }
 
-export function hasFilterHandler(filters, filter) {
-  if (typeof filter === "undefined") {
+/**
+ * hasFilterHandler
+ * - if testValue is undefined, checks to see if the chart has any active filters
```

Comment:
I *love* these comments. It would take me forever to figure out what was going on without this comment block!